Formula/cycode.rb

```diff
@@ -0,0 +1,129 @@
+class Cycode < Formula
+  include Language::Python::Virtualenv
+
+  desc "CLI boosts SDLC security via secret scanning, SAST, SCA, and IaC"
```

Comment:
```suggestion
  desc "Boost software development security via secret scanning, SAST, SCA, and IaC"
```
We should try to minimise the number of acronyms we use here.